### DISCUSSION TITLE
Most aliases are identified as spam or not real - Any suggestion?

### DISCUSSION THREAD

**Discussion by delaney64** [anonaddy/anonaddy#700] [Q&A]
Pretty self explanatory from the title. Whenever I try to send from an alias o sign up its flagged as not legit.

**Comment by willbrowningme** [comment]
There is a list being put together here - https://github.com/GHolk/addy-io-unofficial/wiki are you able to add the websites there or let me know which sites they are and I'll add them?

Have you tried other alias domains or a custom domain?

**Comment by delaney64** [comment]
Hello,
I am unsure of how to create a domain

On Wed, Nov 27, 2024 at 3:50 AM Will Browning ***@***.***>
wrote:

> There is a list being put together here -
> https://github.com/GHolk/addy-io-unofficial/wiki
> <https://github.com/GHolk/addy-io-unofficial/wiki>
> are you able to add the websites there or let me know which sites they are
> and I'll add them?
>
> Have you tried other alias domains or a custom domain?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/anonaddy/anonaddy/discussions/700#discussioncomment-11392238>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMIIB4JTRMFFCWNX3BX7HIT2CWBWDAVCNFSM6AAAAABSRXVV3WVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMZZGIZDGOA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

Disclaimer

The information contained in this communication from the sender is confidential. It is intended solely for use by the recipient and others authorized to receive it. If you are not the recipient, you are hereby notified that any disclosure, copying, distribution or taking action in relation of the contents of this information is strictly prohibited and may be unlawful.

This email has been scanned for viruses and malware, and may have been automatically archived by Mimecast, a leader in email security and cyber resilience. Mimecast integrates email defenses with brand protection, security awareness training, web security, compliance and other essential capabilities. Mimecast helps protect large and small organizations from malicious activity, human error and technology failure; and to lead the movement toward building a more resilient world. To find out more, visit our website.
